nidm/experiment/Core.py

```diff
@@ -321,8 +321,12 @@ def serializeJSONLD(self):
         #context = context2
 
         context=self.createDefaultJSONLDcontext()
+
         #WIP: LOOK AT https://github.com/satra/nidm-jsonld
-        return rdf_graph_parse.serialize(format='json-ld', context=context, indent=4).decode('ASCII')
+        #return rdf_graph_parse.serialize(format='json-ld', context=context, indent=4).decode('ASCII')
+        g=rdf_graph_parse.serialize(format='json-ld', context=context, indent=4).decode('ASCII')
+        import pyld as ld
+        return ld.jsonld.compact(json.loads(g), context)
```

Comment:
and make sure that:

`context['@version'] = 1.1`

Reply:
So this worked.  My main problem appeared to be that I was setting context['@version'] to the string 1.1 not the floating point number.  Anyway it serializes now...but my tests still fail because pyld needs to be added as a required package.